cyber/common/file.cc

```diff
@@ -163,8 +163,17 @@ std::vector<std::string> Glob(const std::string &pattern) {
 bool CopyFile(const std::string &from, const std::string &to) {
   std::ifstream src(from, std::ios::binary);
   if (!src) {
-    AERROR << "Source path doesn't exist: " << from;
-    return false;
+    AWARN << "Source path could not be normally opened: " << from;
+    std::string command = "cp -r " + from + " " + to;
+    ADEBUG << command;
+    const int ret = std::system(command.c_str());
+    if (ret == 0) {
+      ADEBUG << "Copy Success,Command returns " << ret;
```

Comment:
Refine the sentence: `"Copy success, command returns "`

Reply:
Done